code/datums/tgs_event_handler.dm

```diff
@@ -1,39 +1,53 @@
+/// Will our restart be slower?
+GLOBAL_VAR_INIT(slower_restart, FALSE)
+
+/proc/server_announce_global(announcement_text)
+	to_chat(world, "<span style='color: #ff00ff'>\[Server] [announcement_text]</span>")
+
+/proc/server_announce_adminonly(announcement_text)
+	to_chat(GLOB.admins, "<span style='color: #ff00ff'>\[Server]</span> <span class='adminsay'>\[Admin] [announcement_text]</span>")
```

Comment:
Yes I know this also sends messages to mentors, but quite frankly thats not an issue. Mentors are fine to know if the server is going to have an update, and PRRs are also included in this list so :shrug: